environments/prod/prod.tfvars

```diff
@@ -931,6 +950,7 @@ frontends = [
     www_redirect     = true
   },
   {
+    product          = "feespay"       
```

Comment:
can these have - in them? if not it should be stripped out,

products should really map to this list:
https://github.com/hmcts/cnp-jenkins-config/blob/master/team-config.yml

Reply:
Yeah can add "-" if required, they are in naming format as such:-
hmcts-shutter-reform etc

Reply:
This looks very inconsistent in the team-config file